lib/download.js

```diff
@@ -37,3 +37,22 @@ async function findStarknetFoundryDir(extractedPath) {
     `could not find Starknet Foundry directory in ${extractedPath}`,
   );
 }
+
+export async function downloadUniversalSierraCompiler() {
+  const { exec } = require("child_process");
+
+  const command =
+    "curl -L https://raw.githubusercontent.com/software-mansion/universal-sierra-compiler/master/scripts/install.sh | sh";
```

Comment:
I'm not sure using curl here is a way to go actually, I suppose doing something like this here https://github.com/foundry-rs/setup-snfoundry/blob/main/lib/download.js might be a better approach (but I'm no js expert though, so 🤷 )